### DISCUSSION TITLE
Github Copilot query

### DISCUSSION THREAD

**Discussion by buruguduystunstugudunstuy** [community/community#43305] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hi,

We would like to inquire which (Copilot) plan is suited for our needs. What we require is a corporate account wherein we can assign licenses for our users, just like how we assign licenses to our M365 users.  Any prerequisites?

Exhausted all forms to send Github Copilot sales or support.  Hopefully community, sales or support personnel may reachout.  

Thanks and regards,

Vincent

**Comment by iamcjwilliams** [comment]
@buruguduystunstugudunstuy Have you submitted an inquiry with our copilot for business sales team here? : https://github.com/enterprise/contact?ref_page=/pricing?_features=site_copilot_biz_ga&ref_cta=Contact%20Sales&ref_loc=cards

**Comment by buruguduystunstugudunstuy** [comment, marked as answer]
Yes,

Thank you for reaching out, yes, that's the one.  Someone already reached out to me...after several hours.

😄

Appreciate it.

Regards,

Vincent
________________________________
